anchore_engine/analyzers/hints.py

```diff
@@ -64,15 +64,19 @@ def __init__(self, pkg):
     def resolve_rpm_fields(self):
         from anchore_engine.util.rpm import split_rpm_filename
 
+        if self.release:
```

Comment:
Do we need to check for the existence of the other three class members used here? Are they always guaranteed to exist?

Reply:
@dspalmer99 `self.arch` has a default of `x86_64` so I don't think that needs to be checked for. `self.name` and `self.version` are required fields per [the docs](https://docs.anchore.com/current/docs/overview/concepts/images/analysis/content_hints) and they are confirmed to exist [earlier in the code](https://github.com/anchore/anchore-engine/blob/master/anchore_engine/analyzers/hints.py#L23).

Reply:
Thanks, gotcha